src/backends/cg.rs

```diff
@@ -264,6 +272,124 @@ impl<D: HasDisplayHandle, W: HasWindowHandle> SurfaceInterface<D, W> for CGImpl<
         })
     }
 
+    fn new_with_alpha(window_src: W, _display: &D) -> Result<Self, InitError<W>> {
```

Comment:
Should forward to `new` instead of duplicating function

Reply:
Fixed 👍